reana-serial-steps.yaml

```diff
@@ -0,0 +1,25 @@
+version: 0.3.0
```

Comment:
You can directly modify `reana.yaml` as it looks better with step names than without.